docs/ja/docs/tutorial/encoder.md

```diff
@@ -0,0 +1,34 @@
+# JSON互換エンコーダ
+
+データ型（Pydanticモデルのような）をJSONと互換性のあるもの（`dict`や`list`など）に変更する必要がある場合があります。
+
+例えば、データベースに保存する必要がある場合です。
+
+そのために、**FastAPI** は`jsonable_encoder()`関数を提供しています。
+
+## `jsonable_encoder`の使用
+
+JSON互換のデータのみを受信するデータベース`fase_db`があるとしましょう。
```

Comment:
"fake_db" in original sentence.
(Probably its's typo and I know its's a mere example db name.)